core/domain/exp_services.py

```diff
@@ -1218,11 +1218,11 @@ def get_updated_committer_ids(
 
 def get_updated_version_history_model(
     exploration: exp_domain.Exploration,
-    change_list: List[exp_domain.ExplorationChange],
+    change_list: Sequence[exp_domain.ExplorationChange],
     committer_id: str,
     old_states: Dict[str, state_domain.State],
     old_metadata: exp_domain.ExplorationMetadata
-) -> exp_models.ExplorationVersionHistoryModel:
+) -> Optional[exp_models.ExplorationVersionHistoryModel]:
```

Comment:
Don't understand this ... how can it be None? Also the docstring isn't updated to explain this case.